### PR TITLE
Adding version env variable

### PR DESCRIPTION
**Pull request by meda123**
Adding the NEW_RELIC_METADATA_SERVICE_VERSION=XXXXXXXX variable. DO NOT publish until I send a confirmation to the help doc channel.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Review by nbaenam — DISMISSED** [review]
Thank you for the update! I've added some minor modifications.

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx [hunk 1]
```diff
@@ -4108,6 +4108,47 @@ For tips on configuring logs for the Node.js agent, see [Configure Node.js logs
   </Collapser>
 </CollapserGroup>
 
+## Version tracking [#version-tracking]
+
+
+Setting `NEW_RELIC_METADATA_SERVICE_VERSION` creates a tag, `tag.service.version` on event data. This tag is the version of your deployed code, in many cases a semantic version such as 1.2.3, but not always. Sending this information allows you to facet your telemetry by the version of the software deployed, so you can quickly identify which versions of your software are producing the errors.
```
Comment (jmartin4563) [inline src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx:4114]: From the Node.js agent engineers:

We have concerns adding this documentation for an environment variable that currently does not exist in our agent code. The way this is written it appears as if this is something that customers can actually configured and use, while there have been zero code changes at the agent level, nor has this change even been discussed/prioritized with us. If our intent is to "preview" functionality in our docs for upcoming changes, we would advocate that it not be part of the configuration page, and recommend that it is reworded to explicitly state that this is future functionality.

cc @coreyarnold

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx [hunk 2]
```diff
@@ -4338,7 +4379,7 @@ This section defines two configuration options only available with environment v
     id="home"
     title="NEW_RELIC_HOME"
   >
-    Path to the directory containing `newrelic.js`. This is available only as an environment variable. You cannot set it in your config file.
+    Path to the directory containing `newrelic.js`. This is available only as an environment variable. You cann't set it in your config file.
```
Comment (jmartin4563) [inline src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx:4382]:
```suggestion
    Path to the directory containing `newrelic.js`. This is available only as an environment variable. You can't set it in your config file.
```

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx [hunk 3]
```diff
@@ -4371,7 +4412,7 @@ This section defines two configuration options only available with environment v
   >
     If used, this prevents the agent from reading configuration settings from `newrelic.js`. Default values and values from environment variables will still be set.
 
-    This is available only as an environment variable. You cannot set it in your config file.
+    This is available only as an environment variable. You cann't set it in your config file.
```
Comment (jmartin4563) [inline src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx:4415]:
```suggestion
    This is available only as an environment variable. You can't set it in your config file.
```

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx [hunk 4]
```diff
@@ -4108,6 +4108,47 @@ For tips on configuring logs for the Node.js agent, see [Configure Node.js logs
...
+            Type
+          </th>
+
+          <td>
+            String
+          </td>
+        </tr>
+
+        <tr>
+          <th>
```
Comment (jmartin4563) [inline src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx:4137]: Should we also include here that the default is nothing, like the other environment variables specify their defaults?

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx (hunk 3 above)
Comment (nbaenam) [reply src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx]: Good catch, thank you!

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx (hunk 4 above)
Comment (nbaenam) [reply src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx:4137]: I think we should if it's more clear. I'm going to add it.

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx (hunk 2 above)
Comment (nbaenam) [reply src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx]: Good catch! Thank you!

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx (hunk 3 above)
Comment (nbaenam) [reply src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx]: Good catch! Thank you!

**Review by nbaenam — APPROVED** [review]

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx [hunk 5]
```diff
@@ -4108,6 +4108,57 @@ For tips on configuring logs for the Node.js agent, see [Configure Node.js logs
...
+            [Environ variable](#environment)
+          </th>
+
+          <td>
+            `NEW_RELIC_METADATA_SERVICE_VERSION`
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
```
Comment (obarney) [inline src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx:4157]: It looks like the pattern is to put the text here not above the collapser

**Review by obarney — COMMENTED** [review]
@meda123 it looks like your branch contains a whole bunch of unrelated changes so I am wondering if it might need to have the latest changes from develop merged into it.

**Comment by akristen** [comment]
Confirmed with @meda123 that this PR was handled already. Am closing + archiving from our Draft board!

**Pull request closed without merging**